datacube/drivers/postgis/_api.py

```diff
@@ -1474,33 +1500,38 @@ def remove_lineage_relations(self,
     def temporal_extent_by_prod(self, product_id: int) -> tuple[datetime.datetime, datetime.datetime]:
         query = self.temporal_extent_full().where(Dataset.product_ref == product_id)
         res = self._connection.execute(query)
-        return res.first()
+        tmin, tmax = res.first()
+        return (self.time_min.normalise_value(tmin), self.time_max.normalise_value(tmax))
 
     def temporal_extent_by_ids(self, ids: Iterable[DSID]) -> tuple[datetime.datetime, datetime.datetime]:
         query = self.temporal_extent_full().where(Dataset.id.in_(ids))
         res = self._connection.execute(query)
-        return res.first()
+        tmin, tmax = res.first()
+        return (self.time_min.normalise_value(tmin), self.time_max.normalise_value(tmax))
+
+    time_min = DateDocField('aquisition_time_min',
```

Comment:
I realize you are moving existing code, but are the names fixed because of legacy, or could the missing `c` in acquisition be added?

Reply:
We only access the fields by order above, so the names are completely arbitrary.  :)